app/uk/gov/hmrc/valuationofficeagencycontactfrontend/controllers/SatisfactionSurveyController.scala

```diff
@@ -20,19 +20,26 @@ import javax.inject.{Inject, Singleton}
 import play.api.Logger
 import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.Action
+import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import uk.gov.hmrc.valuationofficeagencycontactfrontend.connectors.AuditingService
 import uk.gov.hmrc.valuationofficeagencycontactfrontend.{FrontendAppConfig, Navigator}
 import uk.gov.hmrc.valuationofficeagencycontactfrontend.views.html.{confirmation, satisfactionSurveyThankYou}
 import uk.gov.hmrc.valuationofficeagencycontactfrontend.controllers.actions.{DataRequiredAction, DataRetrievalAction}
 import uk.gov.hmrc.valuationofficeagencycontactfrontend.forms.{SatisfactionSurvey, SatisfactionSurveyForm}
 import uk.gov.hmrc.valuationofficeagencycontactfrontend.utils.{AddressFormatters, DateFormatter, UserAnswers}
 
+import scala.concurrent.ExecutionContext
+
 @Singleton()
 class SatisfactionSurveyController @Inject()(val appConfig: FrontendAppConfig,
                                              val messagesApi: MessagesApi,
                                              navigator: Navigator,
                                              getData: DataRetrievalAction,
-                                             requireData: DataRequiredAction) extends FrontendController with I18nSupport {
+                                             requireData: DataRequiredAction,
+                                             auditService: AuditingService) extends FrontendController with I18nSupport {
+
+  implicit val hc: HeaderCarrier = HeaderCarrier()
```

Comment:
hc can't be delared as `val`. It's case class which is different for every request. Maybe this can help : 


```suggestion
  implicit def hc(implicit request: Request[_]):HeaderCarrier = HeaderCarrierConverter.fromHeadersAndSession(request.headers, request.session)
```